formidable/forms/__init__.py

```diff
@@ -113,10 +114,34 @@ class MyForm(FormidableForm):
     """
 
     @classmethod
-    def to_formidable(cls, label, description=u''):
-        form = Formidable.objects.create(label=label, description=description)
+    def to_formidable(cls, label=None, description=None, instance=None):
+        if not instance:
+            if not label:
+                raise ValueError("On creation mode, label value is mandatory.")
```

Comment:
I'm wondering if this error shouldn't be a `KeyError` instead of `ValueError`? Not sure, what do you think?
Also, having "label" and "value" in that message is a bit confusing. I would go with something like "label is required on creation mode".